DOC.md

```diff
@@ -977,10 +979,13 @@ is only a short outline, their usage is shown more expansively in
 `Examples/snippets.lua`):
 
 - `lambda`: A shortcut for `functionNode`s that only do very basic string-
-manipulation. For example, to replace all occurences of "a" in the nth insert
-with "e", one could use `lambda(lambda._1:gsub("a", "e"), n)` (signature is
-similar to that of `functionNode`).
-If a node has multiple lines, they will be concatenated using "\n".
+  manipulation.  
+  For example, to generate text where all occurences of "a" in the text of the
+  node with jump-index `n` are replaced with "e", one could use
+  `lambda(lambda._1:gsub("a", "e"), n)` (signature is similar to that of
+  `functionNode`).  
+  `lambda._n` returns the text of the `n`th argnode. If this text has multiple
```

Comment:
does argnode here refer to jump-index too?

Reply:
Oh, this is a bit subtle:
`l(l._1 .. l._2, {2,1})` (`l` as `lambda`), would return the text from the node with jump-index **2** followed by that of **1**.
So the order of argnodes only depends on the order of that list (second argument), not their jump-index.
But this could definitely be more clear, I'll reference the args passed to function/dynamicNode I think.

Reply:
Oh, I see what you mean. So this actually refers to the table indexes within the user_arguments argument passed to the lambda (which is the third argument in a normal function). Definitely very specific

Reply:
Ah, not `user_arguments`, that's for more easily re-using similar functions.
`functionNode(fn, argnode-references, opts)`, and `fn` is `function(argnode-text, snippet, user_args)`
(where `argnode_references` could be a list of `jump-indices` (or just a single one)), and `l(lambda-expression, argnode-references)`